flips/20220811-fungible-tokens-metadata.md

```diff
@@ -0,0 +1,210 @@
+# Fungible Tokens Metadata
+
+| Status        | Draft       |
+:-------------- |:---------------------------------------------------- |
+| **FLIP #1087**    | [1087](https://github.com/onflow/flow/pull/1087)|
+| **Author** | Álvaro Lillo Igualada (alvaro.lillo@dapperlabs.com) |
+| **Sponsor**   | Josh Hannan (joshua.hannan@dapperlabs.com)             |
+| **Updated**   | 2022-08-16                                           | 
+
+## Objective
+
+This proposal, in a similar fashion as the NFT Metadata, will make possible to create generic solutions that will interoperate through views rather than through hard-coded types. This will allow better interoperability between fungible tokens and external applications. 
```

Comment:
This paragraph currently mentions the *how* (views), but does not describe the goal. Maybe:
```suggestion
The goal of this proposal is to allow Fungible Tokens (FT) to provide metadata, in a similar fashion as the [NFT Metadata Standard, FLIP 636](https://github.com/onflow/flow/blob/master/flips/20210916-nft-metadata.md). This will allow better interoperability between fungible tokens and external applications. 
```